js/base-tool.js

```diff
@@ -549,6 +549,163 @@ function baseTool () {
 				});
 			},
 		},
+		{
+			name: "Mass-Modify Pages",
+			desc: "Quickly modify multiple pages.",
+			html: `
+				<div id="d20plus-mass-page-modify" title="Better20 - Mass-Modify Pages">
+					<div id="modify-pages-list">
```

Comment:
avoid using `id` except on the outermost part of the window (the `id="d20plus-mass-page-modify"`)

everything inside that should use semantic form identifiers, e.g. `name="..."`

Reply:
Mass-Delete Pages uses id for their list. Can we keep mine too? Or change both of them to name?

Reply:
yeah, there's plenty of old junk code kicking about

If you want to fix the old stuff to use `name`, then go for it